up42/workflow.py

```diff
@@ -468,6 +470,82 @@ def _helper_run_job(
             job.track_status()
         return job
 
+    def _helper_run_parallel_job(
+        self,
+        input_parameters_list: List[Dict] = None,
+        max_concurrent_jobs: int = 10,
+        test_job=False,
+        name: str = None,
+    ) -> "JobCollection":
+        """
+        Helper function to create and run parallel real or test jobs.
+
+        Args:
+            input_parameters_list: List of dictionary of input parameters.
+            max_concurrent_jobs: Maximum number of parallel jobs that can be triggered.
+            test_job: If set, runs a test query (search for available imagery based on your data parameters).
+            name: The job name. Optional, by default the workflow name is assigned.
+
+        Returns:
+            The spawned real or test job object.
+        """
+        if input_parameters_list is None:
+            raise ValueError(
+                "Select the job_parameters, use workflow.construct_parallel_parameters()!"
+            )
```

Comment:
So you mean that it won't be possible that someone set it more than 10? I hardcoded to 10 now because I am not sure if someone can put it more. I guess it makes more sense if it is less than 10 I should throw an error (for now) and then if it is possible to have beyond 10 concur... job then I will remove the explicit 10.

Reply:
I should maybe just check for now if the max number of concurrent job is set to 10 (Since I hardcoded it)... But I am not sure ...

Reply:
Ah I see, then resolved. I though this was exposed to the user and wanted to give an error message in case the user puts in more than 10. But that doesn't make sense, should always be 10 and not exposed.